tasks.py

```diff
@@ -0,0 +1,5 @@
+from dmdevtools.invoke_tasks import library_tasks as ns
```

Comment:
Ah, I should document this somewhere. `ns` is a magic name for invoke, if it finds a `Collection` called `ns` or `namespace` in `tasks.py` it will use that for the list of tasks for that directory.

Reply:
Link to invoke documentation elucidating further http://docs.pyinvoke.org/en/stable/concepts/namespaces.html#starting-out